### PR TITLE
fix bug does not filter data

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24668

## Description
CB integration does not filter data on "!cbd-find-events" command.

### CODE REVIEW

File: Packs/CarbonBlackDefense/pack_metadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-    "Endpoint"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
-}
+    "name": "Carbon Black Defense",
+    "description": "Next-generation antivirus + EDR in one cloud-delivered platform that stops commodity malware, advanced malware, non-malware attacks and ransomware.",
+    "support": "xsoar",
+    "currentVersion": "1.1.0",
```
Comment (ShahafBenYakir) [inline Packs/CarbonBlackDefense/pack_metadata.json:5]: Why is it minor?

File: Packs/CarbonBlackDefense/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+- __Carbon Black Defense__
+ -Fix filter data with "sha256Hash" on "! cbd-find-events" command.
```
Comment (ShahafBenYakir) [inline Packs/CarbonBlackDefense/ReleaseNotes/1_1_0.md:4]: Was this autogenerated?

File: Packs/CarbonBlackDefense/Integrations/integration-CarbonBlackDefense.yml [hunk 3]
```diff
@@ -1054,7 +1054,7 @@ script:
     - name: ipAddress
       description: Query parameter. Filter on events generated by a device with a
         given external or internal IP address
-    - name: sha256hash
+    - name: sha256Hash
```
Comment (ShahafBenYakir) [inline Packs/CarbonBlackDefense/Integrations/integration-CarbonBlackDefense.yml:1057]: Why is this needed? This is very dangerous is it can break backward compatibility.

**Comment by kirbles19** [comment]
@moishce Done
